src/MainLayout.tsx

```diff
@@ -20,7 +20,7 @@ const LoadableStudioLayout = loadable(() => import('./views/studio/StudioLayout'
   ),
 })
```

Comment:
```tsx
const LoadableStudioLayout = loadable(() => import('./views/studio/StudioLayout'), {
  resolveComponent: (components) => components.StudioLayout,
  fallback: (
    <>
      <TopbarBase variant="studio" />
      <StudioLoading />
    </>
  ),
})
```
I think it should do it.
Of course, you'll have to update `src/views/studio/index.ts`and `src/views/studio/StudioLayout.tsx`

Reply:
Oh, my mistake, this will output: 
![image](https://user-images.githubusercontent.com/51168865/122036345-bf86ad00-cdd3-11eb-85a2-bc374b11907b.png)
let's leave it as it is.